packages/ui/.storybook/storybook-main.scss

```diff
@@ -6,4 +6,5 @@
 //@import "../components/vuestic-sass/global/normalize.scss";
 @import './assets/icons/icons.scss';
 @import './assets/fonts.scss';
+@import './assets/global';
```

Comment:
Can we prefer tailwind? We will forget about this approach and external contributors will use a different way.

@asvae, correct me if we're not allowed to use tailwind in stories.

Reply:
I removed tailwind helpers that were in the original stories as it was stated as one of the refactoring steps in #3676

> Try not to use tailwind helpers - we want to remove it from storybook eventually.

@asvae, need your say here on what is the latest agreement on this

Reply:
Tailwind helpers are all right. One concern might be not overusing them.

I.e. where it's enough to use only one helper - don't use 3.

And where it's enough to use `<p>` - don't use helpers at all.

Reply:
I changed guide a bit to
> Please use tailwind for helpers, but not overuse them. Minimum amount of styling is appreciated.

Reply:
Returned tailwind helpers 🙌